doc/mgr/orchestrator.rst

```diff
@@ -107,6 +107,25 @@ Many hosts can be added at once using
 
 This can be combined with service specifications (below) to create a cluster spec file to deploy a whole cluster in one command.  see ``cephadm bootstrap --apply-spec`` also to do this during bootstrap. Cluster SSH Keys must be copied to hosts prior to adding them.
 
+.. _orchestrator-host-labels:
+
+Host labels
+-----------
+
+The orchestrator supports assigning labels to hosts. Labels
```

Comment:
Can you have more than one label? "label**s**" would seem to imply as much.

Reply:
That's correct. Added a sentence.